Bash autocomplete panic

**Describe the bug**
I used `waypoint -autocomplete-install` to enable auto-completion in Bash. When i trigger the auto-completion i get the following error message:
```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/hashicorp/waypoint/internal/cli.logger(0xc00004c080, 0x4, 0x4, 0x3, 0xc00099a700, 0x2, 0x4, 0x5, 0x34a1c93, 0x0, ...)
	gopath/src/github.com/hashicorp/waypoint/internal/cli/main.go:427 +0x57a
github.com/hashicorp/waypoint/internal/cli.Main(0xc00004c080, 0x4, 0x4, 0x0)
	gopath/src/github.com/hashicorp/waypoint/internal/cli/main.go:63 +0x92
main.main()
	gopath/src/github.com/hashicorp/waypoint/cmd/waypoint/main.go:14 +0xa5
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/hashicorp/waypoint/internal/cli.logger(0xc000134040, 0x4, 0x4, 0x3, 0xc00093e7c0, 0x2, 0x4, 0x7, 0x34a7ffb, 0x0, ...)
	gopath/src/github.com/hashicorp/waypoint/internal/cli/main.go:427 +0x57a
github.com/hashicorp/waypoint/internal/cli.Main(0xc000134040, 0x4, 0x4, 0x0)
	gopath/src/github.com/hashicorp/waypoint/internal/cli/main.go:63 +0x92
main.main()
	gopath/src/github.com/hashicorp/waypoint/cmd/waypoint/main.go:14 +0xa5
```

**Steps to Reproduce**
1. Run `waypoint -autocomplete-install`
2. Open new Terminal
3. Type `waypoint` followed by the Tab-Key

**Expected behavior**
No Error.

**Additional context**
Tested with release `v0.1.1` and git commit 2711c261

--- a/internal/cli/main.go
+++ b/internal/cli/main.go
@@ -424,7 +424,7 @@ func logger(args []string) ([]string, hclog.Logger, io.Writer, error) {
 	// This overrides whatever the env var set.
 	var outArgs []string
 	for _, arg := range args {
-		if arg[0] != '-' {
+		if len(arg) != 0 && arg[0] != '-' {
 			outArgs = append(outArgs, arg)
 			continue
 		}

